### PR TITLE
chore(utils): remove duplicate log

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A

**Summary**
<h3><ins>Before</ins></h3>
before

<h3><ins>After</ins></h3>
after

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

File: packages/utils/src/modify-config-helper.ts [hunk 1]
```diff
@@ -139,11 +139,6 @@ export function modifyHelperUtil(
                     },
                     finalConfig,
                 ) as TransformConfig;
-                if (finalConfig.usingDefaults && finalConfig.usingDefaults === true) {
-                    const runCommand = getPackageManager() === 'yarn' ? 'yarn build' : 'npm run build';
-
-                    logger.log(`\nYou can now run ${green(runCommand)} to bundle your application!\n`);
```
Comment (snitin315) [inline packages/utils/src/modify-config-helper.ts:145]: > `'green' is defined but never used `
> `'getPackageManager' is assigned a value but never used`

Can we fix these warnings?

Comment (jamesgeorge007) [reply packages/utils/src/modify-config-helper.ts:145]: Yup

**Review by snitin315 — APPROVED** [review]

**Review by anshumanv — DISMISSED** [review]
add tests?

**Comment by jamesgeorge007** [comment]
> add tests?

We already have adequate integration tests for `init`.

**Comment by anshumanv** [comment]
> We already have adequate integration tests for init.

We clearly don't have test to check we're not logging twice else this wouldn't have happened

**Comment by evilebottnawi** [comment]
Yep we should avoid decrease coverage

File: test/init/auto/init-auto.test.js [hunk 2]
```diff
@@ -48,5 +50,10 @@ describe('init auto flag', () => {
             expect(pkgJson['scripts']['build'] == 'webpack').toBeTruthy();
         };
         expect(pkgJsonTests).not.toThrow();
+
+        // Check we are not logging twice
+        if (process.platform !== 'win32') {
```
Comment (snitin315) [inline test/init/auto/init-auto.test.js:55]: you can use `isWindows` util here.

File: test/init/generator/init-inquirer.test.js [hunk 3]
```diff
@@ -42,5 +44,10 @@ describe('init', () => {
             expect(pkgJson['scripts']['build'] == 'webpack').toBeTruthy();
         };
         expect(pkgJsonTests).not.toThrow();
+
+        // Check we are not logging twice
+        if (process.platform !== 'win32') {
```
Comment (evilebottnawi) [inline test/init/generator/init-inquirer.test.js:49]: We should stop to adding test with skipping on widows, what is the problem here?

Comment (jamesgeorge007) [reply test/init/generator/init-inquirer.test.js:49]: The respective assertion fails on windows; need to investigate why that happens.

Comment (evilebottnawi) [reply test/init/generator/init-inquirer.test.js:49]: Please look at this, it is real bad when we skip tests on windows

**Review by evilebottnawi — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Pull request merged**